operators/Makefile

```diff
@@ -321,31 +327,20 @@ e2e-local:
 
 ci: dep-vendor-only check-fmt lint generate check-local-changes unit integration e2e-compile docker-build
 
-# Run e2e tests in a dedicated gke cluster, that we delete if everything went fine.
-# Let's use n1-standard-8 machine to have enough room for multiple pods on a single node.
-ci-e2e: ci-bootstrap-gke e2e
-
-# Run e2e tests in gke using custom operator image
-ci-e2e-rc: export SKIP_DOCKER_COMMAND=true
-ci-e2e-rc: ci-bootstrap-gke e2e
+# Run e2e tests in a dedicated cluster.
+ci-e2e: deployer
+	$(MAKE) IMG_SUFFIX=-ci install-crds apply-psp e2e
 
-ci-bootstrap-gke:
-	PSP=1 GKE_MACHINE_TYPE=n1-standard-8 $(MAKE) bootstrap-gke
+deployer: dep-vendor-only
```

Comment:
`deployer` feels a bit abstract for someone that would discover this Makefile for the first time.
Would maybe deserve a comment or a more explicit name (`deploy-k8s-cluster`?)

Reply:
The issue with indicating what deployer will do is that it depends on the run-config present. I'll add a comment and change it to run-deployer. WDYT? I can be more explicit when I'll be modifying dev-facing targets.